frame/benchmarking/src/lib.rs

```diff
@@ -791,6 +794,20 @@ macro_rules! impl_benchmark {
 								"Read/Write Count {:?}", read_write_count
 							);
 
+							let time = $crate::benchmarking::current_time();
```

Comment:
ahh this is so good 👍

Reply:
It is called outside of the timed benchmark section